src/utils/globalVars.ts

```diff
@@ -28,6 +28,6 @@ const fieldNames: GroupInfoFieldNames = {
   email: "Email",
   startTime: "Date de début",
   endTime: "Date de fin",
-};
+} as const;
```

Comment:
Try out using `as const` for types like these.  You can super good intellisense.  And because TS uses duck typing, it'll still behave like `GroupInfoFieldNames` anywhere that that's required.